### PR TITLE
Extract all the fields for a Kinesis Record and not just the metadata

### PR DESCRIPTION
**Pull request by ph**
Add data field which contains raw data of the events and information
about the partition, the sequence and the schema.

Add a test to ensure the tranformation is OK from a KinesisRecord to a
beat.Event.

### CODE REVIEW

**Comment by ph** [comment]
need to be backported to 6.7, 7.0 and 7.x

File: x-pack/functionbeat/provider/aws/transformer/transformer.go [hunk 1]
```diff
@@ -64,13 +64,17 @@ func KinesisEvent(request events.KinesisEvent) []beat.Event {
...
+				"event_name":              record.EventName,
+				"event_source":            record.EventSource,
+				"event_source_arn":        record.EventSourceArn,
+				"event_version":           record.EventVersion,
+				"aws_region":              record.AwsRegion,
+				"message":                 string(record.Kinesis.Data),
+				"kinesis_partition_key":   record.Kinesis.PartitionKey,
+				"kinesis_schema_version":  record.Kinesis.KinesisSchemaVersion,
+				"kinesis_sequence_number": record.Kinesis.SequenceNumber,
+				"kinesis_encryption_type": record.Kinesis.EncryptionType,
```
Comment (urso) [inline x-pack/functionbeat/provider/aws/transformer/transformer.go]: How about having a link to the kinesis event documentation for reference in this functions docstring?

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -112,10 +112,10 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Functionbeat*
 
 - Mark Functionbeat  as GA. {pull}10564[10564]
-
 - Correctly normalize Cloudformation resource name. {issue}10087[10087]
 - Functionbeat can now deploy a function for Kinesis. {10116}10116[10116]
 - Allow functionbeat to use the keystore. {issue}9009[9009]
+- Correctly extract Kinesis Data field form the Kinesis Record. {pull}11141[11141]
```
Comment (urso) [inline CHANGELOG.next.asciidoc]: s/form/from/

**Comment by ph** [comment]
I will make the change and merge them right away and backport it.

**Pull request merged**